src/components/notifications/ConfigurationTab.tsx

```diff
@@ -202,6 +239,86 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
             )
     }
 
+    renderWebhookConfigurationTable() {
+        if (this.state.view === ViewType.LOADING) {
+            return (
+                <div className="flex" style={{ height: 'calc(100% - 68px)' }}>
+                    <Progressing pageLoader />
+                </div>
+            )
+        } else if (this.state.webhookConfigurationList.length === 0) {
+            return (
+                <div style={{ height: 'calc(100% - 70px)' }}>
```

Comment:
same here

Reply:
inline css used for particular piece of code, not used anywhere else

Reply:
still move it to scss file

Reply:
done